molgenis-app-manager/src/main/java/org/molgenis/app/manager/controller/AppController.java

```diff
@@ -63,7 +63,7 @@ public AppController(AppManagerService appManagerService, UserPermissionEvaluato
 	public ModelAndView serveApp(@PathVariable String pluginId, Model model, HttpServletRequest request,
 			HttpServletResponse response) throws IOException
 	{
-		PluginIdentity pluginIdentity = new PluginIdentity(pluginId);
+		PluginIdentity pluginIdentity = new PluginIdentity("app/" + pluginId + "/");
```

Comment:
variable name pluginId is too confusing here
app/{uri}/ = pluginid

Reply:
changed to appName